src/core/egg-herbie.rkt

```diff
@@ -104,14 +104,14 @@
 
   ; node -> natural
   ; inserts an expression into the e-graph, returning its e-class id.
+
```

Comment:
Not a big deal but better to avoid making unnecessary whitespace changes like this.